pkg/ovsdb/ovsdb.go

```diff
@@ -261,47 +239,18 @@ func (s *ServOVSDB) Unlock(ctx context.Context, param interface{}) (interface{},
 //  "result": <table-updates2>
 //  "error": null
 //  "id": same "id" as request
-func (s *ServOVSDB) Monitor_cond(ctx context.Context, param []interface{}) (interface{}, error) {
+func (s *ServOVSDB) Monitor_cond(ctx context.Context, param ovsjson.CondMonitorParameters) (interface{}, error) {
 	fmt.Printf("Monitor_cond %T %+v\n", param, param)
 
-	resp, err := s.dbServer.GetData("ovsdb/"+param[0].(string), false)
-	if err != nil {
-		return nil, err
-	}
+	return s.getMonitoredData(param.DatabaseName, param.MonitorCondRequests)
 
-	databases := Databases{Database: map[string]ovsjson.Initial{}}
-	for _, v := range resp.Kvs {
-		keys := strings.Split(string(v.Key), "/")
-		in, ok := databases.Database[keys[3]]
-		var id InitialData
-		if !ok {
-			in = ovsjson.Initial{}
-			databases.Database[keys[3]] = in
-		}
-		if in.Initial == nil {
-			in.Initial = InitialData{}
-		}
-		id = in.Initial.(InitialData)
-		switch keys[5] {
-		case "name":
-			id.Name = string(v.Value)
-		case "connected":
-			id.Connected, _ = strconv.ParseBool(string(v.Value))
-		case "leader":
-			id.Leader, _ = strconv.ParseBool(string(v.Value))
-		case "model":
-			id.Model = string(v.Value)
-		case "schema":
-			id.Schema = string(v.Value)
-		}
-		databases.Database[keys[3]] = ovsjson.Initial{Initial: id}
-	}
-	return databases, nil
 }
 
-func (s *ServOVSDB) Monitor_cond_change(ctx context.Context, param interface{}) (interface{}, error) {
+func (s *ServOVSDB) Monitor_cond_change(ctx context.Context, param []interface{}) (interface{}, error) {
```

Comment:
Name should be MonitorCondChange

Reply:
no, it cannot, 
It should be inline with OVSD protocol.